src/consumer.cpp

```diff
@@ -118,6 +118,8 @@ void IceflowConsumer::subscribeToTopicPartition(uint64_t topicPartition) {
                           std::placeholders::_1));
 
     m_subscriptionHandles.push_back(subscriptionHandle);
+
+    std::cout << "Subscribed to " << dataId << std::endl;
```

Comment:
Should this use a logging statement instead?

Reply:
No, this debug message is quite crucial to know which topic consumer is subscribing to

Reply:
Hmm, but this is a part of the library itself, so I think this should not be printed to the standard output.

```suggestion
    NDN_LOG_INFO("Subscribed to " << dataId);
```